ElectronGunAnalysis/analysis/checkFitSyst.cpp

```diff
@@ -87,7 +86,7 @@ int main( int argc, char* argv[] ) {
         gr_nofit->SetPoint( gr_nofit->GetN(), thisX, thisY );
 
         h1_thisSyst->Fill( (thisBaseline->Eval(thisX)-thisY) );
-        h1_systTot ->Fill( (thisBaseline->Eval(thisX)-thisY) );
+        h1_systTot ->Fill( (thisBaseline->Eval(thisX)-thisY)*1000. ); //in pA
```

Comment:
non mi piace che metti solo uno dei due in pA. mettili entrambi, anzi assicurati che tutto il programma sia in pA, coerentemente.